Update to Laravel 6 & 7, + PHP 7.4 compatible

lauhakari opened the pull request:
This might be nice to have.

What I could notice no big changes and tests passed.

lauhakari committed:
Update to be Laravel 6 compatible.

ben182 commented:
Addressing #94

mnvx commented:
Travis fails on PHP 5.6. Maybe will be better to just remove 5.6 from Travis config, because Laravel 6 [supports PHP 7.2+](https://github.com/laravel/laravel/blob/c70c986e58fe1a14f7c74626e6e97032d4084d5f/composer.json#L11)

lauhakari committed:
Update .travis.yml

Updated php-versions.

lauhakari committed:
Update FindCommand.php

Fixed PHP 7.4  compatibility, for: ErrorException: Unparenthesized `a ? b : c ? d : e` is deprecated

lauhakari committed:
Update README.md

Added Travis CI badge.

lauhakari committed:
Update README.md

Added Travis CI badge.

weidmaster commented:
Merging this does fix the issue, as it is only a parenthesis error in the ternary condition. Please accept it so we can update in our composer json. Thank you.

lauhakari committed:
Update with Laravel 7 support.

lauhakari committed:
Reverted custom README changes.

vincentkedison commented:
Please help to release it to packagist.
I use ben182/laravel-auto-translate, and this package depends on themsaid/laravel-langman.
Causing error on php 7.4

```
ErrorException  : Unparenthesized `a ? b : c ? d : e` is deprecated. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)`
```

Thank you
